Documentation/operations/performance/tuning.rst

```diff
@@ -37,6 +37,55 @@ in any of the Cilium pods and look for the line reporting the status for
 * eBPF-based kube-proxy replacement
 * eBPF-based masquerading
 
+IPv6 BIG TCP
+============
+
+IPv6 BIG TCP allows the network stack to prepare larger TSO (transmit) and GRO
```

Comment:
Nit: s/TSO/GSO/ ? I know “TSO” exists, but the rest of the document only mentions GSO (and conversely, it uses GSO later without mentioning it's for transmitting).

Reply:
That was intentional actually because GSO segments are later passed down for TSO, but I agree it's best to be consistent.
I'll change it, thanks!